pkg/apis/operator/v1alpha1/tektonresult_types.go

```diff
@@ -75,13 +73,6 @@ type ResultsAPIProperties struct {
 	TLSHostnameOverride   string `json:"tls_hostname_override,omitempty"`
 	AuthDisable           bool   `json:"auth_disable,omitempty"`
 	AuthImpersonate       bool   `json:"auth_impersonate,omitempty"`
-	S3BucketName          string `json:"s3_bucket_name,omitempty"`
-	S3Endpoint            string `json:"s3_endpoint,omitempty"`
-	S3HostnameImmutable   bool   `json:"s3_hostname_immutable,omitempty"`
-	S3Region              string `json:"s3_region,omitempty"`
-	S3AccessKeyID         string `json:"s3_access_key_id,omitempty"`
-	S3SecretAccessKey     string `json:"s3_secret_access_key,omitempty"`
-	S3MultiPartSize       int64  `json:"s3_multi_part_size,omitempty"`
```

Comment:
@khrm removing fields from TektonResults CR can break the existing installation
Have you added in breaking section about that?
If by mistake user upgraded to this version with existing result CR, Is it possible to remove the results CR?

Reply:
s3 fields can be added as secrets as per your earlier PR. DB fields were redundant. They aren't used anywhere.

Reply:
I mean DB user and password are always taken from secret.

Reply:
@khrm Yes, I understand that part. The problem is,
* assume the user already has installed results with removed options(in this PR) in their environment
* now they upgraded to the latest version with this patch

I assume in this case operator/results webhook will fail the tektonresults modification, will report unknown fields

Reply:
Yes. That's why I have requested Misha that they remove the existing TektonResults CR and recreate it.